oak_containers_launcher_server/src/lib.rs

```diff
@@ -43,6 +53,8 @@ struct LauncherServerImplementation {
     system_image: std::path::PathBuf,
     container_bundle: std::path::PathBuf,
     application_config: Option<std::path::PathBuf>,
+    // Attestation Evidence is initialized by the Orchestrator.
+    attestation_evidence: Mutex<Option<AttestationEvidence>>,
```

Comment:
This looks like a OnceCell could be used here, especially since we only expect this to be set once.

Reply:
In fact, as @andrisaar suggested, if RefCell works, that may be the ideal option. But it may not work.
Have you tried that already?

Reply:
`RefCell` cannot be shared between threads (and the code does have them because of gRPC)
```
error[E0277]: `RefCell<Option<AttestationEvidence>>` cannot be shared between threads safely
  --> oak_containers_launcher_server/src/lib.rs:61:19
   |
61 | impl Launcher for LauncherServerImplementation {
   |                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `RefCell<Option<AttestationEvidence>>` cannot be shared between threads safely
```

Reply:
thanks for trying it out and letting us know, that's useful